### PR TITLE
Release process fixes [BT-501]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: processes/release_processes/README.MD [hunk 1]
```diff
@@ -60,7 +60,7 @@ Ensure you have at least 8GB; 4GB is not sufficient.
 
 #### Let people know the publish is underway
 
-Post another message in `#dsp-batch` that the release is underway, asking everyone to hold off merges to develop until
+Post another message in `#ftfy-private` that the release is underway, asking everyone to hold off merges to develop until
```
Comment (mcovarr) [inline processes/release_processes/README.MD:63]: as discussed in standup

File: processes/release_processes/firecloud-develop.dot [hunk 2]
```diff
@@ -42,7 +42,6 @@ digraph {
     dspjenkins_merge [shape=oval label="Merge dsp-jenkins PR"];
 
     jenkins_set [shape=oval label="[Jenkins] Use dsl-seed to make our dsp-jenkins branch the default"];
-    jenkins_reset [shape=oval label="[Jenkins] Use dsl-seed to make 'master' the default again"];
```
Comment (mcovarr) [inline processes/release_processes/firecloud-develop.dot:45]: This step no longer appears to be necessary and apparently was recently removed from the DAG but not the DOT file so it was confusingly rendered as a dissociated bubble with no arrows in or out.

File: processes/release_processes/README.MD (hunk 1 above)
Comment (jgainerdewar) [reply processes/release_processes/README.MD:63]: Thanks for catching this 🤦‍♀️

**Review by jgainerdewar — APPROVED** [review]

File: processes/release_processes/firecloud-develop.dot (hunk 2 above)
Comment (breilly2) [reply processes/release_processes/firecloud-develop.dot:45]: 🤔  I hope that was an intentional change.

**Review by breilly2 — APPROVED** [review]

File: processes/release_processes/firecloud-develop.dot (hunk 2 above)
Comment (mcovarr) [reply processes/release_processes/firecloud-develop.dot:45]: Yes the `dsl-seed` job seems to run automatically to reset to `master` when the branch in `dsp-jenkins` is merged.

**Pull request merged**